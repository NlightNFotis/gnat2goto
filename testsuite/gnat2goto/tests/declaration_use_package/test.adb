with Lib;
procedure Test is
   procedure Use_Lib is
      use Lib;
   begin
      A;
   end Use_Lib;
begin
   Use_Lib;
end Test;
