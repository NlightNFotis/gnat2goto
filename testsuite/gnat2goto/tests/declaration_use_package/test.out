Error from cbmc test:
**** WARNING: no body for function lib__a

VERIFICATION SUCCESSFUL
